### PR TITLE
[#678] Detail page of comingsoon APIs shows API info

### LINKED ISSUE #678: API pages marked as "coming soon" should not be indexed
**Describe the bug**

Right now API pages marked as "Coming Soon" are indexed and returned via search bar. Try https://developers.italia.it/it/api/trasporti-opendata.html for example. This page should not be indexed and should not show up.

**To Reproduce**
1. Go to Search Bar
2. Search for "opendata"
3. Click the result
4. You are seeing a page that should not be seen

**Expected behavior**

Those pages should not be indexed and visible.

### PR DESCRIPTION
**Pull request by ccatterina**
## Description

As suggested by @bfabio https://github.com/italia/developers.italia.it/issues/678#issuecomment-704068256, the PR makes the detail page of comingsoon APIs display API info and comingsoon badge instead of the error message "No API definition provided".

Result:
image

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

## Fixes

- Fixes #678

### CODE REVIEW

**Review by bfabio — APPROVED** [review]
:rocket:

File: _layouts/api-details.html [hunk 1]
```diff
@@ -26,4 +26,14 @@
 
 {% assign swaggerProjects = filteredProjects | jsonify | escape %}
 
-{% include swagger.html swaggerUrl=page.openapi logo=page.contact.logo lang=active_lang l10n=l10n swaggerProjects=swaggerProjects %}
+{% if page.comingsoon != true %}
```
Comment (bfabio) [inline _layouts/api-details.html:29]:
```suggestion
{% unless page.comingsoon %}
```

File: _layouts/api-details.html [hunk 2]
```diff
@@ -26,4 +26,14 @@
...
+  {% include swagger.html swaggerUrl=page.openapi logo=page.contact.logo lang=active_lang l10n=l10n swaggerProjects=swaggerProjects %}
+{% else %}
+  <div class="container my-5">
+    <h1 class="d-inline-block">
+      {{ page.title }}
+      <span class="badge badge-pill badge-secondary align-top h5">{{ t.coming_soon }}</span>
+    </h1>
+    <h2 class="subtitle">{{ page.abstract }}</h2>
+  </div>
+{% endif %}
```
Comment (bfabio) [inline _layouts/api-details.html:39]:
```suggestion
{% endunless %}
```

**Comment by ccatterina** [comment]
I added the suggestions and squashed the commits into the first one.

**Review by sebbalex — APPROVED** [review]
🚀

**Pull request merged**